DATA/common/setenv_calib.sh

```diff
@@ -91,6 +100,10 @@ if [[ -z $CALIBDATASPEC_BARREL ]]; then
     if [[ $CALIB_TOF_LHCPHASE == 1 ]] || [[ $CALIB_TOF_CHANNELOFFSETS == 1 ]]; then add_semicolon_separated CALIBDATASPEC_BARREL "calibTOF:TOF/CALIBDATA/0"; fi
     if [[ $CALIB_TOF_DIAGNOSTICS == 1 ]]; then add_semicolon_separated CALIBDATASPEC_BARREL "diagWords:TOF/DIAFREQ/0"; fi
 
+    # TPC
+    if [[ $CALIB_TPC_TIMEGAIN == 1 ]]; then add_semicolon_separated CALIBDATASPEC_BARREL "tpcmips:TPC/MIPS"; fi
+    if [[ $CALIB_TPC_RESPADGAIN == 1 ]]; then add_semicolon_separated CALIBDATASPEC_BARREL "trackGainHistoTPC:TPC/TRACKGAINHISTOS"; fi
```

Comment:
If this is used for the `o2-dpl-raw-proxy`, shouldn't it be
```suggestion
    if [[ $CALIB_TPC_TIMEGAIN == 1 ]]; then add_semicolon_separated CALIBDATASPEC_BARREL "tpcmips:TPC/MIPS/0"; fi
    if [[ $CALIB_TPC_RESPADGAIN == 1 ]]; then add_semicolon_separated CALIBDATASPEC_BARREL "trackGainHistoTPC:TPC/TRACKGAINHISTOS/0"; fi
```
Or is it for the `o2-dpl-output-proxy`, then I think it is fine.

Reply:
Hi @wiechula , 

Indeed, someone told me not long ago that one of the two proxies might have needed the subSpec, but I was not sure (you confirm that this is the case, only for one proxy?). Now I added it. 

Cheers,

Chiara

Reply:
Btw, the spec is used both for the output and raw proxy.